src/formats/jsonLd.ts

```diff
@@ -105,7 +112,7 @@ function fixQuads(plainQuads: any[]): Quad[] {
   return fixedQuads;
 }
 
-function term(term: any) {
+function term(term: Quad_Object | DefaultGraph) {
```

Comment:
Why not just `RDF.Term` (https://github.com/rdfjs/types/blob/183bda795f57a9464ddf95deac45a0c4a48879cf/data-model.d.ts#L13)? The only difference in the typings is that `RDF.Term` allows the slightly looser `RDF.BaseQuad` rather than enforcing `RDF.Quad` (https://github.com/rdfjs/types/blob/183bda795f57a9464ddf95deac45a0c4a48879cf/data-model.d.ts#L152); but I don't see the problem anyways since there is an error thrown in either case.

Reply:
If RDF.Term works, I'm happy to have a PR to improve on it; the types in that file are all a little weird. I'd probably be in favour of a standalone `@inrupt/solid-parsers` project that pulls in jsonld and turtle and makes them work fully.

Reply:
> the types in that file are all a little weird

How so, I've never had problems working with these type interfaces in the past?

> @inrupt/solid-parsers project

I would note that https://github.com/rubensworks/rdf-parse.js/blob/master/package.json achieves that though it is a little more heavy weight as it supports a number of other serializations.

Otherwise would recommend switching to https://github.com/rubensworks/jsonld-streaming-parser.js which wont require the Term conversion seen here.

Reply:
Have created a ticket for this, and will continue without it.